meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-boot.mount

```diff
@@ -7,7 +7,7 @@ Conflicts=umount.target
 [Mount]
 What=/dev/disk/by-label/resin-boot
 Where=/mnt/boot
-Type=vfat
+Type=ubifs
```

Comment:
We can't change the fs type to ubifs in meta-resin, we'll have to figure out something else here.